velox/common/file/File.cpp

```diff
@@ -142,21 +142,60 @@ uint64_t LocalReadFile::preadv(
   // Dropped bytes sized so that a typical dropped range of 50K is not
   // too many iovecs.
   static thread_local std::vector<char> droppedBytes(16 * 1024);
+  uint64_t totalBytesRead = 0;
   std::vector<struct iovec> iovecs;
   iovecs.reserve(buffers.size());
+
+  auto preadvRun = [&]() -> ssize_t {
+    auto bytesRead = folly::preadv(fd_, iovecs.data(), iovecs.size(), offset);
+    if (bytesRead < 0) {
+      perror("preadv");
+      LOG(ERROR) << "preadv failed with error: " << strerror(errno);
```

Comment:
Remove perror("preadv");? Is it required?
```
LOG(ERROR) << "preadv failed with error: " << folly::errnoStr(errno);
```

Reply:
Removed.